### DISCUSSION TITLE
绳网为什么要关啊啊啊啊啊T_T

### DISCUSSION THREAD

**Discussion by ChaoKaiADG** [share121/inter-knot#5499] [公告]
@share121作者我求你了别关行吗
全网最纯净的米家讨论平台
为什么说关就关啊T_T

**Comment by shidijun** [comment]
哥 首先谁说要关的 我看作者也没得动态啊
其次 不知道你用的是不是HTML反正你多少写错了

**Reply by Yizakl** [reply]
![IMG_0152](https://github.com/user-attachments/assets/a781388e-03a9-409c-8646-3d58b454764f)

**Reply by ChaoKaiADG** [reply]
看作者在绳网上发的帖子，说有法律风险，yao关了

**Comment by hjejdjdj** [comment]
确实
